### ISSUE TITLE
Allow to set an icon to display on loading screen instead of Frigate logo

### ISSUE THREAD

**Issue by felipecrs** [dermotduffy/frigate-hass-card#990]
While the Frigate logo is amazing, I don't think their colors fits my dashboard well.

![image](https://user-images.githubusercontent.com/29582865/222933173-89855f16-2b74-4c6b-9eaf-9e5b6799eb9c.png)

If I had the option to choose, I would choose displaying an MDI icon centered on the screen.

I could just build an image with the icon myself, upload it and set the card to use its URL, but I having this option would be better since it would not require to download any image.

**Comment by dermotduffy** [comment]
Just trying to understand the usecase: Why is the bird image showing up there? Is that screensaver mode?

**Comment by felipecrs** [comment]
I honestly don't understand either, but it does show up occasionally.

There is one specific situation in which I can make it show up consistently, which is when I exit the editor mode for example.

But I'm not in home right now so I can't really share details.

My assumption is that, perhaps, it shows up whenever fetching the images from HA fails?

**Comment by dermotduffy** [comment]
> My assumption is that, perhaps, it shows up whenever fetching the images from HA fails?

Yes. It will show up when it is asked to render an image, but no other image is available for whatever reason. Sounds like it is in these temporary cases you'd like to be able to show something else. If yes -- got it.

**Comment by bagobones** [comment]
The behavior in v5 beta1 should be different if you are using the go2rtc provider and don't have the show still image option set while loading.

For birdseye you need to set a [Custom Birdseye Icon](https://docs.frigate.video/configuration/birdseye/#custom-birdseye-icon) in frigate to get ride of its black and white logo.

**Comment by BasicCPPDev** [comment]
Using this card with frigate on an proxmox lxc container. While frigate is running flawlessly i got that logo flashing every 5 min. On my dashboard for my 3 camera card at same time. Not sure if i have to open another specific issue for this case.

I though it was due to my fully kiosk tablet using wifi but its not.

I tried a couple of changes like setting go2rtc or webrtc but i didn't got any result.

So what can cause this every 5 min?

**Comment by felipecrs** [comment]
To be honest, I have a feeling that the stream restarts in the Frigate card on every 5 minutes indeed. Let me double check it.

**Comment by dermotduffy** [comment]
+1, please open a new issue if you can reproduce this as it does not seem at all related to this feature request. Thanks.
